content/general/concepts/data-mining/data-mining.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Data Mining'
+Description: 'Data mining is the process of applying algorithms to search for patterns within collections of data.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+  - 'Machine Learning'
+Tags:
+  - 'Data Science'
+  - 'Deep Learning'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+**Data mining** is the process of applying algorithms to search for patterns within collections of data.
+
+Fundamentally, data mining is the deployment of an automated process for analyzing large amounts of data that otherwise could not be addressed manually. This process generally involves several steps that may include data collection, data cleaning and validation (to address any errors, inconsistencies, and other quality issues), model development and testing, and final deployment.
+
+In this context model development means the application of one or more of several machine learning (ML) algorithms such as regression, decision trees, support vector machines, as well as many other tools that are part of the modern ML repertoire.
+
+## Context
+
+Data mining is one of the more recent applications of statistical analysis for extracting information from data. Within the scientific and social domains there has always been a need and a refinement of efforts in the collection and processing of data. Previously, data collection was a largely manual effort, as was the processing of this data. However, with modern computers, and the ubiquity of computers in many of our devices, we now have many more data creation and processing possibilities than ever before. One of the most prevalent and challenging issues for businesses and individuals alike is the abundance of data. The modern affliction is now, amongst all that is available, what is relevant and worthwhile, and what does it mean.
```

Comment:
```suggestion
Data mining is one of the more recent applications of statistical analysis for extracting information from data. Within the scientific and social domains, there has always been a need and a refinement of efforts in the collection and processing of data. Previously, data collection was a largely manual effort, as was the processing of this data. However, with modern computers, and the ubiquity of computers in many of our devices, we now have many more data creation and processing possibilities than ever before. One of the most prevalent and challenging issues for businesses and individuals alike is the abundance of data. The modern affliction is now, amongst all that is available, what is relevant and worthwhile, and what does it mean.
 ```

Reply:
@caupolicandiaz Adding on to this, can the third sentence be changed to remove the words "we" and "our", which we typically try to avoid using in Docs entries.

Reply:
Definitely, good catch.